internal/services/network/network_manager_network_group_resource.go

```diff
@@ -99,6 +107,10 @@ func (r ManagerNetworkGroupResource) Create() sdk.ResourceFunc {
 				group.Properties.Description = &model.Description
 			}
 
+			if model.MemberType != "" {
```

Comment:
since `member_type` has a default value, will it be an empty value?

Reply:
fixed